.examples/docker-compose-in-memory-config/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3.3"
+services:
+  gatus:
+    image: twinproduction/gatus
+    ports:
+      - 8080:8080
+    environment:
+      GATUS_CONFIG: > 
+        {endpoints: [{name: website, url: 'https://twin.sh/health', interval: 5m, conditions: ['[STATUS] == 200', '[BODY].status == UP', '[RESPONSE_TIME] < 300']}, {name: example, url: 'https://example.org/', interval: 60s, conditions: ['[STATUS] == 200']}]}
```

Comment:
Is there a reason why you decided to go with this approach? 
IIRC, you should be able to have multi-line strings by using `|` instead of `>`:
```yaml
version: "3.3"
services:
  gatus:
    image: twinproduction/gatus
    ports:
      - 8080:8080
    environment: |
      endpoints:
        # ...
```

Reply:
I modified the docker-compose file as per your suggestion.
The example now shows a configuration using standard multi-line YAML.
See commit f7bdf71c05e67c42754a7c2c330154691ba4468d